### ISSUE TITLE
Unity

### ISSUE THREAD

**Issue by KennyBarraud** [NeoLSN/cordova-plugin-gyroscope#3]
Hi,
I wanted to knew what is the unity of the vector3 returned by the getCurrent function.
Thanks for that plugin :3

**Comment by JosonCai** [comment]
Can't understand your meaning. You mean use the engine Unity or other?

**Comment by gutosantos82** [comment]
No, he probably meant the measure/scale: meters, seconds, degrees.
Gyroscope is angular velocity so maybe something like degrees/sec

**Comment by jaxonPetersen94** [comment]
Please reword question for better results! We do not know what is being asked. Thank you !
